package.json

```diff
@@ -11,6 +11,8 @@
     }
   ],
   "devDependencies": {
+    "@teppeis/doctrine": "^3.2.0",
+    "@types/esprima": "^4.0.2",
```

Comment:
How is this different from the very similar-looking "esprima": "^4.0.1" below?

Reply:
`@types/esprima` only provides type annotations for the esprima package and allows type checking of the ts generator itself. The actual implementation of esprima still only lives in the esprima package, not in this one.